mmdet/datasets/pipelines/test_time_aug.py

```diff
@@ -37,10 +36,10 @@ def __init__(self,
         if not self.flip and self.flip_direction != ['horizontal']:
             warnings.warn(
                 'flip_direction has no effect when flip is set to False')
-        if (self.flip and
-                not any([isinstance(_, RandomFlip) for _ in self.transforms])):
+        if (self.flip
+                and not any([t.type == 'RandomFlip' for t in transforms])):
```

Comment:
For `dict` indexing, we may use `t['type']` for general usage.

Reply:
thanks. I have updated it.